neo/Consensus/ConsensusContext.cs

```diff
@@ -200,6 +241,20 @@ public ConsensusPayload MakePrepareRequest()
             });
         }
 
+        public ConsensusPayload MakeRecoveryMessage()
+        {
+            return MakeSignedPayload(new RecoveryMessage()
+            {
+                PrepareMsgWitnessInvocationScripts = PreparationWitnessInvocationScripts,
+                ChangeViewWitnessInvocationScripts = ChangeViewWitnessInvocationScripts,
+                Nonce = Nonce,
+                NextConsensus = NextConsensus,
+                TransactionHashes = TransactionHashes,
+                MinerTransaction = (MinerTransaction)Transactions[TransactionHashes[0]],
```

Comment:
This can cause NRE now, since we can create recovery messages after a change view but before PrepareResponse has been received. I'll account for that.

Reply:
Fixed now.